sdk/servicebus/service-bus/src/serviceBusMessage.ts

```diff
@@ -389,14 +384,36 @@ export function toRheaMessage(
   //   amqpMsg.message_annotations![Constants.viaPartitionKey] = msg.viaPartitionKey;
   // }
 
-  if (msg.scheduledEnqueueTimeUtc != null) {
-    amqpMsg.message_annotations![Constants.scheduledEnqueueTime] = msg.scheduledEnqueueTimeUtc;
-  }
+  updateScheduledTime(amqpMsg, msg.scheduledEnqueueTimeUtc);
 
   logger.verbose("SBMessage to RheaMessage: %O", amqpMsg);
   return amqpMsg;
 }
 
+/** @internal @ignore */
+export function updateMessageId(rheaMessage: RheaMessage, messageId: RheaMessage["message_id"]) {
+  if (messageId != null) {
+    if (typeof messageId === "string" && messageId.length > Constants.maxMessageIdLength) {
+      throw new Error(
+        "Length of 'messageId' property on the message cannot be greater than 128 characters."
```

Comment:
Nit: I know you didn't write this error message, but feels like we should reference the constant here rather than hard-code the number 128:
```suggestion
        `Length of 'messageId' property on the message cannot be greater than ${Constants.maxMessageIdLength} characters.`
```